docs/v1.0/in_forward.txt

```diff
@@ -246,6 +246,55 @@ Multiple messages may be sent in the same connection.
 
 For more details, see [Fluentd Forward Protocol Specification (v1)](https://github.com/fluent/fluentd/wiki/Forward-Protocol-Specification-v1).
 
+## Tips & Tricks
+
+### How to enable TLS/SSL encryption
+
+Since v0.14.12, Fluentd includes a built-in TLS/SSL support. Here we present a quick up and running tutorial for setting up TLS/SSL encryption:
+
+First, generate a self-signed certificate using the following command:
```

Comment:
Can we use `fluent-ca-generate` command here?

Reply:
I do not believe that the current distribution of td-agent includes the script. So many users cannot use the generation script.

BTW I can confirm that the stand-alone installation (a.k.a. through Ruby gem) of Fluentd includes the `fluent-ca-genrate` script. I do not know the reason behind the difference.

Reply:
It depends on @repeatedly 's ts-agent3 packaing work.

Reply:
OK.

For now I'll merge this PR as is. If the `fluent-ca-generate` script get included
in feature, I'll update the tutorial.